recipes/goatools/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+    home: 'https://github.com/tanghaibao/goatools'
+    summary: "Python scripts to find enrichment of GO terms"
```

Comment:
license is BSD, sorry have not spotted this earlier. If you submit your PR from the main branch I can fix those for you.